torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -3020,6 +3020,40 @@ RegisterOperators reg2({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(int[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<int64_t> l = pop(stack).toIntList();
+          push(stack, std::move(l));
+          for(int i = 0; i < l.size(); i++){
+            if(l[i] == 0){
+              push(stack, false);
+              break;
+            }
+            else{
+              push(stack, true);
+            }
+          }
+          return 0;
+        },
+        aliasAnalysisFromSchema()),
+    Operator(
+        "aten::all(bool[] self) -> bool",
+        [](Stack& stack) {
+          c10::List<bool> l = pop(stack).toBoolList();
+          push(stack, std::move(l));
+          for(int i = 0; i < l.size(); i++){
+            if(l[i] == 0){
```

Comment:
nit: don't need the `== 0` here

Reply:
Changed this to `!l[i]`. I do a conditional for a false value here as one false value will return false from all() as opposed to checking if every value in the list is true.